build-customization.rst

```diff
@@ -0,0 +1,72 @@
+.. post:: Nov 10, 2022
+   :tags: announcement, feature, builds
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+      We released a new config key (``build.commands``) to specify user-defined commands
+      which allows users to overwrite the build process completely.
+
+
+Overwrite the build process completely
+======================================
+
+We are happy to announce a new *beta* feature that allow users to **overwrite the build process completely**.
+In the past, `we talked about executing custom commands <https://blog.readthedocs.com/user-defined-build-jobs/>`_ *in-between* the Read the Docs build process.
+However, that approach may not be sufficient for some projects with a heavily customized build process,
+some of which are not able to use our platform at all.
+Until now! We have good news for them!
+
+The new key ``build.commands`` allows projects to only execute *exactly* the commands they want.
+No more. No less.
+This means that Read the Docs *won't execute any of the default* commands behind the scenes.
+You have 100% control over the building process.
+
+
+How it works?
+-------------
+
+To use the new *beta* feature,
+you can specify all the commands the build process requires under the ``build.commands`` `config key <https://docs.readthedocs.io/en/stable/config-file/v2.html#build-commands>`_.
+Finally, move the resulting documentation artifacts into the ``_readthedocs/html`` directory.
+Read the Docs will upload and host all the content of that folder ✨
+
+Here is simple example that builds a small project's documentation using `Docsify <https://docsify.js.org/>`_:
```

Comment:
I still feel like this isn't a great example given how different it is than Sphinx. It seems like we're kind of burying the lede here though. It seems like the announcement should be that other tools are now supported, not that you can override our build process completely?

Reply:
I didn't want to communicate the message "other tools are now supported" because they are not integrated yet. That message is going to be in future post after we have the integrations documented.

In this post I want to limit ourselves to communicate there is a new config option that allows people to override the build process. Leaving the door open to their imagination about what they can do with this.

We are still in an exploration stage and we are not sure what people want/is going to do with this.

We don't have _great_ examples at https://github.com/readthedocs/meta/discussions/28. Most of them are junky or can be replaced properly by `build.jobs`. Besides, the ones that we have documented in our official docs are Docsify and Pelican, so I think we should choose between one of them.

If the issue is not generating the HTML, we should go with the Pelican one.

Reply:
I changed the introduction a little to mention other doctools and use a Docusaurus example with a link to a live version of the resulting build. Docusaurus looks like a good example here since it has a completely different build process 👍🏼

Reply:
With this change, I think the post is better:

- limitation of the current fixed build process
- presenting a new config key to change that
- use cases for other doctools and projects with customized build processes
- explain how it works
- show a real example
- mention current limitations
- ask for feedback

💯 

Let me know what you think.